app/controllers/posts_controller.rb

```diff
@@ -16,15 +16,19 @@ def index
   end
 
   def owed
-    posts_started = Post.where(user_id: current_user.id).pluck('distinct id')
-    posts_in = Reply.where(user_id: current_user.id).pluck('distinct post_id')
-    ids = (posts_in + posts_started).uniq
-    @posts = Post.where(id: ids).where.not(status: [Post::STATUS_COMPLETE, Post::STATUS_ABANDONED]).where.not(last_user: current_user)
-    @posts = @posts.where.not(status: Post::STATUS_HIATUS).where('tagged_at > ?', 1.month.ago) if current_user.hide_hiatused_tags_owed?
-    @posts = posts_from_relation(@posts.order('tagged_at desc'))
     @show_unread = true
     @hide_quicklinks = true
     @page_title = 'Replies Owed'
+
+    if params[:view] == 'hidden'
+      ids = PostAuthor.where(user_id: current_user.id, can_owe: false).group(:post_id).pluck(:post_id)
+      @posts = posts_from_relation(Post.where(id: ids)) and return
```

Comment:
I think it's intended, so you can opt yourself back in without needing to currently owe a tag.

Reply:
yeah, that makes sense, just mildly unintinuitive. Might want to have the header specify that somehow? Both so that people don't think it's a bug and so it's more likely to occur to people they can do that.